liquidator/liquidator.js

```diff
@@ -46,6 +46,27 @@ class Liquidator {
 
     let liquidationPromises = []; // store all promises to resolve in parallel later on.
     for (const position of underCollateralizedPositions) {
+      // Make sure that the bot has enough inventory/approval to do the liquidation.
+      const liquidation = this.empContract.methods.createLiquidation(
```

Comment:
You can refactor line 83 with `liquidationPromises.push(liquidation.send(...))`

Reply:
Oops, nice catch. Will do.